src/app/config/components/config.component.ts

```diff
@@ -81,9 +86,8 @@ export class ConfigComponent implements OnInit, OnDestroy {
     }
 
     localStorage.removeItem("base_config");
-
-    this.configApolloService.updateConfig(JSON.stringify(updateDict)).pipe(first()).subscribe(o => {
-      if (!o?.data?.updateConfig) {
+    this.organizationApolloService.changeOrganization(this.organization.id, JSON.stringify(updateDict.limit_checks)).pipe(first()).subscribe((o: any) => {
```

Comment:
i dont think this is correct. only for the limit we need to change the organization, the rest is still done via the config server